### PR TITLE
fix: readd autopause

### LINKED ISSUE #229: Accidentally deleted autopause
### 📝 Description

I have no clue how this got through #223, but I accidentally deleted `/autopause` from `serenity.rs`.

### 🪜 Reproduction Steps

...

### ℹ Environment / Computer Info

- **Parrot version**: `v1.5.0`
- **Operating System**: `Windows 11`

### 📸 Screenshots

_No response_

### PR DESCRIPTION
**Pull request by aquelemiguel**
Fixes #229.
This one is just embarrassing, please approve and let's move on. 🤡

### CODE REVIEW

**Comment by aquelemiguel** [comment]
CI blocking merge because of #231. Once that is merged, pull main into into here and retry.

**Review by joao-conde — APPROVED** [review]

**Pull request merged**
